specs/source.go

```diff
@@ -39,6 +39,10 @@ type Source struct {
 
 	// Backend is the name of the state backend to use
 	Backend Backend `json:"backend,omitempty"`
+	// Enable incremental syncing for tables that supports that
+	Incremental Incremental `json:"incremental,omitempty"`
+	// Sync on
+	OnlyIncrementalTables bool `json:"only_incremental_tables,omitempty"`
```

Comment:
It would be hard to determine which tables are incremental and generate a config and keep it up to date. This way you can have one config with all tables, fetched daily, and another one, fetched hourly, updating only incrementals (taking shorter bursts, not wasting API limits and other resources)

Reply:
True, but I think keeping your list of tables up-to-date (which to include, which to skip) and how regularly you want to sync each is a problem separate from incremental-ness, adding this one boolean is not going to solve it except maybe for a narrow use case